### PR TITLE
Fix symbols for filename (SHA)

### PR DESCRIPTION
**Pull request by alphaX86**
Signed-off-by: Aadhitya A <aadhitya864@gmail.com>

**Description**

This PR fixes the SHA filename problem. Needs testing though!!!

**Notes for Reviewers**
Tests and checks needed in review

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by alphaX86** [comment]
@leecalcote @Revolyssup I just need a way to test this because the updateDocs check is always skipped and I'm unsure if this will work

**Comment by Revolyssup** [comment]
> @leecalcote @Revolyssup I just need a way to test this because the updateDocs check is always skipped and I'm unsure if this will work

If you have tested the script by creating a file in windows with some filename that this new script will create, things should be fine. We will know after the merge

**Comment by alphaX86** [comment]
> > @leecalcote @Revolyssup I just need a way to test this because the updateDocs check is always skipped and I'm unsure if this will work
> 
> If you have tested the script by creating a file in windows with some filename that this new script will create, things should be fine. We will know after the merge

Well, the filename will indeed work.. I just needed to see if the SHA variable works in the CI action now... that's why

**Comment by Revolyssup** [comment]
@alphaX86  Can you use double quotes instead of single quotes?

File: .github/workflows/e2etest-servicemeshandaddon.yaml [hunk 1]
```diff
@@ -95,7 +95,7 @@ jobs:
           echo $pretty > data.json
           cat data.json
           touch test.md
-          filename=$(date '+_${SHORT_SHA}')
+          filename=$(date + '_${SHORT_SHA}')
```
Comment (leecalcote) [inline .github/workflows/e2etest-servicemeshandaddon.yaml:98]:
```suggestion
          filename=$(date + "_${SHORT_SHA}")
```

File: .github/workflows/e2etest-servicemeshinstall.yaml [hunk 2]
```diff
@@ -95,7 +95,7 @@ jobs:
           echo $pretty > data.json
           cat data.json
           touch test.md
-          filename=$(date '+_${SHORT_SHA}')
+          filename=$(date + '_${SHORT_SHA}')
```
Comment (leecalcote) [inline .github/workflows/e2etest-servicemeshinstall.yaml:98]:
```suggestion
          filename=$(date + "_${SHORT_SHA}")
```

**Comment by alphaX86** [comment]
Any other changes needed folks?

**Comment by alphaX86** [comment]
If this format worked... then I'll open the PRs for other adapter repos too. Only thing I need is that if this works or not after merge...

**Comment by leecalcote** [comment]
> If this format worked... then I'll open the PRs for other adapter repos too. Only thing I need is that if this works or not after merge...

Argh. I missed this message. @meshery/adapter-maintainers please jump in when you can.

**Pull request merged**